va/validation-authority.go

```diff
@@ -501,6 +501,8 @@ func (va *ValidationAuthorityImpl) validate(ctx context.Context, authz core.Auth
 	}
 	logEvent.Challenge = *challenge
 
+	va.stats.TimingDuration(fmt.Sprintf("VA.Validations.%s.%s", challenge.Type, challenge.Status), time.Since(vStart), 1.0)
```

Comment:
Why move this after the problem checks? While the code in between won't introduce any real latency it is somewhat confusing to read now.